go/oasis-test-runner/scenario/e2e/txsource.go

```diff
@@ -29,31 +29,34 @@ const (
 
 	nodeRestartIntervalLong = 2 * time.Minute
 	livenessCheckInterval   = 1 * time.Minute
+	txSourceGasPrice        = 1
 )
 
 // TxSourceMultiShort uses multiple workloads for a short time.
 var TxSourceMultiShort scenario.Scenario = &txSourceImpl{
-	basicImpl: *newBasicImpl("txsource-multi-short", "", nil),
+	basicImpl: *newBasicImpl("txsource-multi-short", "simple-keyvalue-client", nil),
```

Comment:
Why is this change needed? The scenario does not actually run this client? (Also below.)

Reply:
thanks for noticing, leftover from some of my initial testing :)